test/test_build_test_container.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+import docker
+
+from exasol_script_languages_container_tool.lib.utils.docker_utils import find_images_by_tag
+
+import utils as exaslct_utils
+from exasol_integration_test_docker_environment.testing import utils
+
+
+class BuildTestContainerTest(unittest.TestCase):
```

Comment:
A general remark: I have the impression that the word "test" is used far too often.

At first glance there seem to be the following significantly different semantic domains:
1. The so-called "test container" in opposite to the "database container" with the test container designed to execute the tests verifying the behavior of the Exasol database inside the database container.
2. "test" code in opposite to "production code", with test code designed to verify the validity of the production features, avoid regressions and bugs.
3. "test" as the general purpose of SLC ITDE, i.e. environments created for the purpose of evaluating the Exasol Database inside a Docker Container (or maybe also in other scenarios), potentially in combination with other components that could be in other Docker Containers as well.

In my eyes this situation creates the problem that the names of many (really many) files, classs, attributes, methods, and variables contain the word "test" but for a reader it is not immediately clear which semantic domain this word is intended to link to, see the following examples:
- `class BuildTestContainerTest(unittest.TestCase):`
- `self.test_environment`
- `test/test_build_test_container.py` (3 times "test", 36 percent of all characters in the path)
- `def test_build_test_container(self):`

In summary I propose to use different words for each different semantic domain.

Candidates for the semantic domains listed above could be
1. test code execution, test orchestration, test runner, laboratory, testbench, qa, audit, isolation, QA Container, test logic container, Validation Container
2. "test"
3. evaluation, rating, environment, sandbox (probably used somewhere else already), docker-based, experiment, TestSuite, Product Evaluation Suite, Product Evaluation Docker Environment, QA Docker Environment, Fertigung, Prüf,

Reply:
Please, create a ticket for it, in some cases I agree, in other cases, such as the test suites, they test frameworks used required the word test in specific places to recognize tests, they should run.

Reply:
see https://github.com/exasol/integration-test-docker-environment/issues/320